cmd/podman/validate/args.go

```diff
@@ -86,6 +86,13 @@ func CheckAllLatestAndIDFile(c *cobra.Command, args []string, ignoreArgLen bool,
 		specifiedIDFile = true
 	}
 
+	if c.Flags().Changed("filter") {
+		if argLen > 0 {
+			return errors.New("no arguments are needed with --filter")
```

Comment:
can we make this stronger than needed?  something like `--filter takes no arguments`?